axes/attempts.py

```diff
@@ -62,35 +43,35 @@ def filter_user_attempts(request: HttpRequest, credentials: dict = None) -> Quer
     return AccessAttempt.objects.filter(**filter_kwargs)
 
 
-def get_user_attempts(request: HttpRequest, credentials: dict = None) -> QuerySet:
+def get_user_attempts(request: HttpRequest, credentials: dict = None, attempt_time: datetime = None) -> QuerySet:
     """
-    Get valid user attempts and delete expired attempts which have cool offs in the past.
+    Get valid user attempts that match the given request and credentials.
     """
 
     attempts = filter_user_attempts(request, credentials)
 
-    # If settings.AXES_COOLOFF_TIME is not configured return the attempts
-    cool_off = get_cool_off()
-    if cool_off is None:
+    if settings.AXES_COOLOFF_TIME is None:
+        log.debug('AXES: Getting all access attempts from database because no AXES_COOLOFF_TIME is configured')
```

Comment:
@aleksihakli are those all `log.debug` be removed or updated to log.info?

Reply:
I think that these debug level outputs are useful for people who wish to gain more insight into their application, but are not that useful when running the application in normal production mode, hence the info level.